### DISCUSSION TITLE
Unable to open DynamicConfiguration in OAPServerConfiguration

### DISCUSSION THREAD

**Discussion by saowu** [apache/skywalking#12135] [General]
![image](https://github.com/apache/skywalking/assets/33569619/b72d99dd-7998-4335-90f2-34b89932b5fb)
https://github.com/apache/skywalking-swck/blob/v0.8.0/operator/config/samples/oapserverconfig.yaml
Unable to open DynamicConfiguration in OAPServerConfiguration
![image](https://github.com/apache/skywalking/assets/33569619/a8db1ef6-49b8-4d79-bf63-f7e56f4d94ba)
![image](https://github.com/apache/skywalking/assets/33569619/75bcce0b-d4af-43ac-87d0-3854b7376b54)

**Comment by wu-sheng** [comment]
@dashanji Could you take a look?

**Comment by dashanji** [comment]
Hi @saowu. The file is out of date, could you please refer to the doc  https://github.com/apache/skywalking-swck/blob/v0.8.0/docs/oapserver-configuration.md for detailed configurations?

**Reply by wu-sheng** [reply]
@dashanji Please remove or update the file in the source codes.

**Reply by dashanji** [reply]
Okay.

**Reply by saowu** [reply]
It has taken effect. Thank you all！
